flags/eigendaflags/cli.go

```diff
@@ -137,6 +139,15 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			Category: category,
 			Required: true,
 		},
+		// Flags that are proxy specific, and not used by the eigenda-client
+		// TODO: should we move this to a more specific category, like EIGENDA_STORE?
```

Comment:
the current category "EigenDAClientCategory" looks alright to me

Reply:
Maybe.... but the point is that eigenda-client is the client only in my mind. Like all the other flags directly populate an eigenda-client config. But this one is different.

Reply:
I am fine with either, tbh, EIGENDA_STORE sounds a bit weird to me because store sounds static, whereas retry is action